Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.cpp

```diff
@@ -103,12 +103,16 @@ namespace ROS2::Utils
 
     UrdfAssetMap FindAssetsForUrdf(const AZStd::unordered_set<AZStd::string>& meshesFilenames, const AZStd::string& urdFilename)
     {
+        auto enviromentalVariable = std::getenv("AMENT_PREFIX_PATH");
+        AZ_Error("UrdfAssetMap", enviromentalVariable, "AMENT_PREFIX_PATH is not found.");
```

Comment:
I'd recommend instead of saying AMENT_PREFIX_PATH is not found, here, indicate:
```
Environment variable AMENT_PREFIX_PATH is empty, (CONCEQUENCE), (HOW TO FIX)
```

like
```
Env var AMENT_PREFIX_PATH is not defined or is empty, will not be able to find package resources.  Try setting this variable to point at the folder containing xxxyyyyzzzz
```

Saves you on having to answer support questions :)